platform-includes/performance/traces-sampler-as-sampler/unreal.mdx

```diff
@@ -0,0 +1,49 @@
+```cpp
+UCLASS()
+class USomeTraceSampler : public USentryTraceSampler
+{
+	GENERATED_BODY()
+
+public:
+	virtual bool Sample_Implementation(USentrySamplingContext* samplingContext, float& samplingValue) override
+    {
+        const FString& path =
+            *samplingContext->GetCustomSamplingContext().Find("__HttpPath");
```

Comment:
Can we translate this to something that resonates more with an Unreal Game?

`__HttpPath` is IIRC what we have on ASP.NET Core.

Do we create transactions with any additional context in the SDK that we can use here in the example?


```suggestion
            *samplingContext->GetCustomSamplingContext().Find("StartGame");
```

Reply:
I've made an attempt to "gamify" this example by determining the sample rate based on a match's game mode. Please let me know if that makes sense